src/pspm_cfg/pspm_cfg_run_sf.m

```diff
@@ -49,5 +49,7 @@
                                             'dispsmallwin', ...
                                             'overwrite', ...
                                             'threshold'});
-out = {pspm_sf(model, options)};
+pspm_sf(model, options);
+out = {model.modelfile};
+
```

Comment:
Similar issue to that in `src/pspm_cfg/pspm_cfg_run_dcm.m`

Reply:
See above.